src/consensus/aft/raft.h

```diff
@@ -1250,8 +1250,6 @@ namespace aft
           continue;
         }
 
-        LOG_DEBUG_FMT("Replicating on follower {}: {}", state->my_node_id, i);
```

Comment:
This was printed for each entry in an AE at the start of the batch, rather than being printed when each entry is individually executed.